test/bin/get_rel_version_repo.sh

```diff
@@ -52,15 +52,11 @@ get_current_release_from_sub_repos() {
 	local -r minor="${1}"
 	local -r rhsm_repo="rhocp-4.${minor}-for-rhel-9-${UNAME_M}-rpms"
 
-	# getting version of RPM within a rhocp repo depends on the repo being enabled,
-	# which is done in configure_vm.sh
-	if dnf_repo_is_enabled "${rhsm_repo}"; then
-		local newest
-		newest=$(sudo dnf repoquery microshift --quiet --queryformat '%{version}-%{release}' --repo "${rhsm_repo}" | sort --version-sort | tail -n1)
-		if [ -n "${newest}" ]; then
-			echo "${newest}"
-			return
-		fi
+	local newest
+	newest=$(sudo dnf repoquery microshift --quiet --queryformat '%{version}-%{release}' --repo "${rhsm_repo}" | sort --version-sort | tail -n1)
+	if [ -n "${newest}" ]; then
+		echo "${newest}"
+		return
```

Comment:
@pmtk I think this is the change you've mentioned that we need, please take a look when you have a few minutes.

Reply:
I think so.
BTW. command can be changed to `sudo dnf repoquery --quiet --latest-limit 1 --qf "%{version}-%{release}" --repo "${rhsm_repo}" microshift` - `--latest-limit` instead of `| sort | tail`

Reply:
If that function were to return "version,repo", then `YMINUS2_RELEASE_REPO="$(get_ocp_repo_name_for_version ${YMINUS2_MINOR_VERSION})"` could be `cut -d, -f2` but I'm fine with either